### ISSUE TITLE
[Bug] Pre-migration activity shows in activity logs

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#8463]
The activity log does not have a DB relationship to projects, so old records are appearing as activity on the project IDs we reappropriated when we performed #8083 and #8377 yesterday. 

<img width="714" alt="Screen Shot 2022-02-11 at 2 22 43 PM" src="https://user-images.githubusercontent.com/1463708/153666239-0f6a4627-a4d1-40fd-860e-8df888c6959f.png">

Please delete all activity logs prior to 2/11/22.

**Comment by frankhereford** [comment]
TLDR: I propose the execution of the following SQL command on the production database, and I will await @amenity's approval to do so.

```
DELETE FROM moped_activity_log
WHERE created_at <= '2022-02-10 14:00:00 +00:00';
```

This query intends to remove any activity log entry that was created prior to 8AM Central (2PM UTC) on February 10, 2022, which was the time we opened the maintenance window to roll out Moped 1.0.0. 

I am also proposing to not do this via a PR / migration. Adding this query to the codebase does not advance the app itself in anyway; this is maintenance work that is particular to our own deployment and would be a speed-bump to any other deployments of the moped application, either by us or another transportation department.

@chiaberry @mateoclarke - doing this outside of a migration is a departure from our past policies, so please let me know if you'd like to discuss this or have concerns. I'll be sure to mention it today in the dev sync as well. @chiaberry - thank you for your time this afternoon to bounce around this idea.

**Comment by amenity** [comment]
Sounds good, @frankhereford - go for it

**Comment by mateoclarke** [comment]
![lord-of-the-rings-destroy-it](https://user-images.githubusercontent.com/5697474/157128975-16197209-ca73-4f30-a7ee-d2a21e4e09ee.gif)

**Comment by frankhereford** [comment]
Thanks y'all - The above query has been executed on the production moped database. 615 records were removed.
